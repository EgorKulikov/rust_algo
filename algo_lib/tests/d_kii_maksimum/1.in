11
1 5
1 3
1 7
0 1
0 2
0 3
-1 5
1 10
0 1
0 2
0 3
