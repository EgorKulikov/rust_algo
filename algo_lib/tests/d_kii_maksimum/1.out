7
5
3
10
7
3
